ocpmodels/models/gemnet_oc/gemnet_oc.py

```diff
@@ -880,53 +882,20 @@ def subselect_edges(
             )
         return subgraph
 
-    def generate_graph(self, data, cutoff, max_neighbors):
+    def generate_graph_dict(self, data, cutoff, max_neighbors):
         """Generate a radius/nearest neighbor graph."""
         otf_graph = cutoff > 6 or max_neighbors > 50 or self.otf_graph
 
-        if self.use_pbc:
-            if otf_graph:
-                edge_index, cell_offsets, num_neighbors = radius_graph_pbc(
-                    data, cutoff, max_neighbors
-                )
-            else:
-                edge_index = data.edge_index
-                cell_offsets = data.cell_offsets
-                num_neighbors = data.neighbors
-
-            out = get_pbc_distances(
-                data.pos,
-                edge_index,
-                data.cell,
-                cell_offsets,
-                num_neighbors,
-                return_offsets=False,
-                return_distance_vec=True,
-            )
-
-            edge_index = out["edge_index"]
-            edge_dist = out["distances"]
-            # These vectors actually point in the opposite direction.
-            # But we want to use col as idx_t for efficient aggregation.
-            edge_vector = -out["distance_vec"] / edge_dist[:, None]
-            cell_offsets = -cell_offsets  # a - c + offset
```

Comment:
@AdeeshKolluru we missed this line [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/ocpmodels/models/gemnet_oc/gemnet_oc.py#L912), right?

Reply:
Yes, that looks correct.